src/index.html

```diff
@@ -9,6 +9,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="../src/images/logo.png"
+          alt="Moyo logotype"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
```

Comment:
[checklist p5](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
```